### ISSUE TITLE
Suggestion with Emoji renders question marks

### ISSUE THREAD

**Issue by crabicode** [i-like-robots/react-tags#159]
GDay,

Suggestions containing emoji symbols are rendered with questions marks.

https://codesandbox.io/embed/0ywq1rvkkv

http://www.giphy.com/gifs/TiIg8tDadsXyJERMZK

**Comment by i-like-robots** [comment]
Thank you for reporting this and your great test case and demo but I can't recreate this. I know you can achieve the same effect by splitting the character into its constituent code poins -  `"🇺🇸".split("")` - but I'm not sure how or why this would occur.

To help narrow it down could you tell me:

- What browser and OS are you using?
- Is your OS set to a language other than English?

**Comment by crabicode** [comment]
I'm using MacOS Sierra with English as a primary language.

Have you checked the codepen I mentioned earlier? If you try it right away
you'll notice the emojis will not be displayed correctly.

Surely, emojis with split would strip off some bits.

**Comment by crabicode** [comment]
You may want to use emoji-aware utilities to split the string.

https://codesandbox.io/s/w6j770j15

```js
import split from 'lodash/split';

console.warn(split("🇺🇸 Bad Suggestion", '')); 
// ["🇺🇸", " ", "t", "e", "x", "t"]

console.warn("🇺🇸 Bad Suggestion".split('')); 
// ["�", "�", "�", "�", " ", "t", "e", "x", "t"]
```

**Comment by i-like-robots** [comment]
Very interesting, thank you for clarifying. I'm afraid even using the demo you provided (https://codesandbox.io/embed/0ywq1rvkkv) and following the exact steps shown in your screengrab I cannot recreate this issue on my device. Which browser are you using?

Split is not used anywhere in this component but there are some regexps that do the same if given a blank query: https://github.com/i-like-robots/react-tags/blob/master/lib/Suggestions.js#L5-L15 (IIRC these were copied from https://github.com/LeaVerou/awesomplete).

I suspect skipping these functions when the query is blank will fix it but without being able to recreate this I cannot confirm this.

**Comment by crabicode** [comment]
I've asked my colleague who worked on Windows 10 with en_US locale and he could reproduce the issue. FYI, I'm using the latest Chrome Version 74.0.3729.108 (Official Build) (64-bit) with en_US locale. I tried to reproduce the issue on Safari Version 10.1.2 (12603.3.8) and it behaved a little bit differently but it still caused the problem. Please see https://giphy.com/gifs/iE4AJQK6m8NJPGaKGN

Not sure where else we should look for a problem :(

```js
'🇺🇸 Bad Suggestion'.replace(/(?:)/gi, '<mark>$&</mark>');

// <mark></mark>�<mark></mark>�<mark></mark>�<mark></mark>�<mark></mark> <mark></mark>B<mark></mark>a<mark></mark>d<mark></mark> <mark></mark>S<mark></mark>u<mark></mark>g<mark></mark>g<mark></mark>e<mark></mark>s<mark></mark>t<mark></mark>i<mark></mark>o<mark></mark>n<mark></mark>
```

**Comment by i-like-robots** [comment]
Thanks again for the extra info @crabicode, that confirms my suspicion. #164 should fix this!

**Comment by crabicode** [comment]
@i-like-robots thanks, it's been fixed 👍

**Comment by i-like-robots** [comment]
Released as 5.11.1
